VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/InitialShape.cpp

```diff
@@ -241,8 +241,91 @@ TArray<FInitialShapeFace> CreateDefaultInitialFaces()
 	return InitialFaces;
 }
 
+bool IsCongruentToDefaultInitialShape(const TArray<FInitialShapeFace>& InitialFaces)
+{
+	const TArray<FInitialShapeFace> DefaultInitialFaces = CreateDefaultInitialFaces();
+	check(DefaultInitialFaces.Num() == 1);
+	const TArray<FVector> DefaultVertices = DefaultInitialFaces[0].Vertices;
+	check(DefaultVertices.Num() == 4);
+
+	if (InitialFaces.Num() == DefaultInitialFaces.Num())
+	{
+		const TArray<FVector> Vertices = InitialFaces[0].Vertices;
+
+		if(Vertices.Num() == DefaultVertices.Num())

```

Comment:
Not sure if you knew about this: we're usually running source code through clang-format before committing, either by pre-commit hook or manually. Please check with @bneukom how he has set it up in the case of Vitruvio.

Since you've had a number of pull requests already, I guess there might be a number of unrelated re-formatting changes, so probably be best to a separate formatting commit, so the source code is properly formatted again.

NB, the idea of enforcing .clang-format / strict formatting rules is that you don't have to think about formatting at all :-)

Reply:
Thank you for the comment, I will try to comply with clang standards from now on